test/lib/mayaUsd/render/mayaToHydra/testMaterialXOnNative.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import maya.cmds as cmds
+import mayaUtils
+import fixturesUtils
+import mtohUtils
+import unittest
+
+class TestMaterialXOnNative(mtohUtils.MtohTestCase):
+    _file = __file__
+
+    IMAGEDIFF_FAIL_THRESHOLD = 0.01
+    IMAGEDIFF_FAIL_PERCENT = 0.1
+
+    def verifySnapshot(self, imageName):
+        cmds.refresh()
+        self.assertSnapshotClose(imageName, 
+                                 self.IMAGEDIFF_FAIL_THRESHOLD,
+                                 self.IMAGEDIFF_FAIL_PERCENT)
+
+    @unittest.skipUnless(mtohUtils.checkForPlugin('LookdevXMaya'), "Requires LookDevX Plugin.")
+    def test_MaterialX(self):
+        mayaUtils.openTestScene("testMaterialX", "RedMtlxSphere.ma")
+        self.setBasicCam(2)
+        self.setHdStormRenderer()
+        self.verifySnapshot("RedMtlxSphere.png")
```

Comment:
are we missing to pass IMAGEDIFF_FAIL_THRESHOLD and IMAGEDIFF_FAIL_PERCENT here?

Reply:
No, they are passed from inside verifySnapshot method

Reply:
right, sorry, I missed that. :)